frontend/src/components/notes/Questions.vue

```diff
@@ -15,7 +15,8 @@
     <table class="question-table mt-2">
       <thead>
         <tr>
-          <th>Approved</th>
+          <th>Edit</th>
+          <th>Approveded</th>
```

Comment:
What does that word mean?...

Reply:
LOL. I guess the first -ed means it's passive void and second means it's past tense. It's English, sir, English.

Reply:
It was just a spam edit because the frontend wasn't reloading (or so I thought) and it accidentally got left there :D

Reply:
Oh noticed that you are looking at an outdated file so nevermind :D